pkg/cloud/services/compute/instance.go

```diff
@@ -432,7 +432,7 @@ func (s *Service) getOrCreateRootVolume(eventObject runtime.Object, instanceSpec
 			return nil, fmt.Errorf("exected to find volume %s with size %d; found size %d", name, size, volume.Size)
 		}
 
-		s.scope.Logger().Info("using existing root volume %s", name)
+		s.scope.Logger().Info("Using existing root volume", "rootVolumeName", name)
```

Comment:
Good point. As you say, when we're doing this already it makes sense to be consistent in how we log.

Reply:
Perhaps it might make sense to add object types like "pool" and "listener" to the set. They are used extensively in pkg/cloud/services/loadbalancer/loadbalancer.go, and possibly other files, too.

Reply:
I agree that creating a new taxonomy will help address the consistency issue. During the migration to structured logging, I was confused by these two rules:
- "Do not provide additional context for how the value is used. Don't use `podIP`, do use `IP`." 
- "When names are very ambiguous, try to include context in the label. For example, instead of `key` use `cacheKey` or instead of `version` use `dockerVersion`."

There seems to be no clear instruction about when to include or exclude the context. This lack of clarity can be problematic for contributors who are not familiar with the default/conventional naming style. It would be beneficial to provide more explicit examples and explanations for each rule.

Reply:
@skylerxhu That's a good catch. That text from the kubernetes guidelines would appear to argue against what I'm suggesting. I think we should assume for now that the guidelines are based on collected experience and just follow them until we find a good reason not to.

To be clear, that means:

* A property (e.g. id, name, ip) for *any* object is just called &lt;property>
* Unless the log message involves more than one object, in which case any 'principal subject' is just %lt;property> and other objects have a prefix

e.g.

```
("Created network", "id", "aaaa-bbbb-cccc-dddd", "name", "my-network")

("Created subnet", "id", "0000-1111-2222-3333", "name" "my-subnet", "networkID", "aaaa-bbbb-cccc-dddd", "networkName", my-network")
```

Thoughts?

Reply:
I don't think this is worth getting hung up on and it's impacting work by others, so lets aim to settle on a decision before EOD today.

Reply:
Also in favor of `name` if there is only one object handled within the log message.

Reply:
On the other hand, there is a list of "good keys", and that includes `volumeName`.

Reply:
By the way, can we use past tense here? "Used existing root volume"
Does that make sense?

Reply:
> By the way, can we use past tense here? "Used existing root volume" Does that make sense?

IIRC the guidelines recommend '-ing' when logging before and action. Past tense indicates a completed action.